content/bn/client-server-architecture.md

```diff
@@ -0,0 +1,19 @@
+---
+title: ক্লায়েন্ট-সার্ভার আর্কিটেকচার
+status: Feedback Appreciated
+category: প্রযুক্তি
+---
+
+## এটা কি
+
+একটি ক্লায়েন্ট-সার্ভার আর্কিটেকচারে (Client-Server Architecture), একটি অ্যাপ্লিকেশন তৈরি করে এমন যুক্তি (Code) দুটি বা ততোধিক উপাদানের মধ্যে বিভক্ত হয়: একটি ক্লায়েন্ট যা কাজ করতে বলে (যেমন আপনার ওয়েব ব্রাউজারে চলমান Gmail ওয়েব অ্যাপ্লিকেশন), এবং একটি বা আরও বেশি সার্ভার যা সেই অনুরোধটি পূরণ করে (যেমন, ক্লাউডে Google-এর কম্পিউটারে চলমান “ইমেল পাঠান”(Send Email) পরিষেবা)। এই উদাহরণে, আপনি যে বহির্গামী ইমেলগুলি লেখেন তা ক্লায়েন্ট দ্বারা পাঠানো হয় (আপনার ওয়েব ব্রাউজারে চলছে ওয়েব অ্যাপ্লিকেশন) একটি সার্ভারে (Gmail এর কম্পিউটার, যা আপনার বহির্গামী ইমেলগুলি তাদের প্রাপকদের কাছে ফরোয়ার্ড করে)।
```

Comment:
It would be great if you could rearrange this sentence "একটি ক্লায়েন্ট-সার্ভার আর্কিটেকচারে (Client-Server Architecture), একটি অ্যাপ্লিকেশন তৈরি করে এমন যুক্তি (Code) দুটি বা ততোধিক উপাদানের মধ্যে বিভক্ত হয়: একটি ক্লায়েন্ট যা কাজ করতে বলে (যেমন আপনার ওয়েব ব্রাউজারে চলমান Gmail ওয়েব অ্যাপ্লিকেশন), এবং একটি বা আরও বেশি সার্ভার যা সেই অনুরোধটি পূরণ করে (যেমন, ক্লাউডে Google-এর কম্পিউটারে চলমান “ইমেল পাঠান”(Send Email) পরিষেবা)"